### ISSUE TITLE
The 'VsPackage' did not load correctly

### ISSUE THREAD

**Issue by gnimor** [dotnet/ResXResourceManager#468]
When starting Visual Studio 2022 I get a dialog with title "The 'VsPackage' did not load correctly". A closer look at the ActivityLog shows an entry:

SetSite failed for package [VsPackage]Source: &apos;mscorlib&apos; Description: Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.&#x000D;&#x000A;System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.&#x000D;&#x000A;   at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)&#x000D;&#x000A;   at System.Reflection.RuntimeAssembly.get_DefinedTypes()&#x000D;&#x000A;   at TomsToolbox.Composition.MetadataReader.Read(Assembly assembly) in D:\a\1\s\src\TomsToolbox.Composition\MetadataReader.cs:line 27&#x000D;&#x000A;   at System.Linq.Enumerable.&lt;SelectManyIterator&gt;d__17`2.MoveNext()&#x000D;&#x000A;   at TomsToolbox.Composition.Ninject.ExtensionMethods.BindExports(IKernel kernel, IEnumerable`1 exportInfos) in D:\a\1\s\src\TomsToolbox.Composition.Ninject\ExtensionMethods.cs:line 44&#x000D;&#x000A;   at ResXManager.VSIX.VsPackage.FillCatalog() in D:\a\1\s\src\ResXManager.VSIX\VSPackage.cs:line 201&#x000D;&#x000A;   at System.Threading.Tasks.Task`1.InnerInvoke()&#x000D;&#x000A;   at System.Threading.Tasks.Task.Execute()&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#x000D;&#x000A;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#x000D;&#x000A;   at ResXManager.VSIX.VsPackage.&lt;InitializeAsync&gt;d__15.MoveNext() in D:\a\1\s\src\ResXManager.VSIX\VSPackage.cs:line 102&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#x000D;&#x000A;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#x000D;&#x000A;   at Microsoft.VisualStudio.Shell.AsyncPackage.&lt;&gt;c__DisplayClass20_0.&lt;&lt;Microsoft-VisualStudio-Shell-Interop-IAsyncLoadablePackageInitialize-Initialize&gt;b__1&gt;d.MoveNext()&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#x000D;&#x000A;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#x000D;&#x000A;   at Microsoft.VisualStudio.Threading.JoinableTask.&lt;JoinAsync&gt;d__76.MoveNext()&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.RethrowException(AggregateException e)&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.InternalGetResult(Boolean ignoreUIThreadCheck)

***** Update *****
Just a small addition to my report above. I have installed both VS2019 and VS2022. For both Visual Studio Versions I have the ResXManager extension installed. I have just removed the extension from the VS2019 installation and the issue in VS2022 is gone. It seems it is somehow related to parallel installation of multiple Visual Studio versions with ResXManager installed.

**Comment by tom-englert** [comment]
Did you try this: https://github.com/dotnet/ResXResourceManager/blob/master/Documentation/Topics/Troubleshooting.md?

**Comment by gnimor** [comment]
Yes, but it seems the issue is related that it was installed for both version. For 2022 it was fresh installation but for 2019 it was already installed. After the installation of 2022 the ResxManager shows the error mentioned above.
